modules/engines/engine-odata/src/main/resources/META-INF/services/org.eclipse.dirigible.core.publisher.api.PublisherHandler

```diff
@@ -0,0 +1 @@
+org.eclipse.dirigible.engine.odata2.synchronizer.ODataSynchronizerPublisherHandler	# OData Synchronizer Publisher Handler
```

Comment:
Change the file name to point to the`IPublishHandler` interface:


`org.eclipse.dirigible.core.publisher.api.PublisherHandler `

->

`org.eclipse.dirigible.core.publisher.api.IPublishHandler`